webapp/packages/core-blocks/src/Button.tsx

```diff
@@ -25,7 +26,7 @@ export type ButtonProps = (React.ButtonHTMLAttributes<HTMLButtonElement | HTMLAn
   icon?: string;
   viewBox?: string;
   mod?: ButtonMod;
-  tag?: 'button' | 'a' | 'div';
+  render?: UiKitButtonProps['render'];
```

Comment:
i suggest renaming it later to avoid big diffs in this PR, right now we need to affect the public interface of base components minimally

Reply:
it's the only way to use ariakit button properly, not so much components were touched by that changes